docs/Tutorial.md

```diff
@@ -505,13 +505,50 @@ const App = () => (
 
 When displaying the posts list, the app displays the `id` of the post author. This doesn't mean much - we should use the user `name` instead. For that purpose, set the `recordRepresentation` prop of the "users" Resource:
 
+```diff
+// in src/users.tsx
+import { useMediaQuery } from '@mui/material';
+-import { List, SimpleList, Datagrid, TextField, EmailField } from "react-admin";
++import {
++    List,
++    SimpleList,
++    Datagrid,
++    TextField,
++    EmailField,
++    Show,
++    SimpleShowLayout,
++} from 'react-admin';
+import MyUrlField from './MyUrlField';
+
+//...
+
++export const UserShow = () => {
```

Comment:
The tutorial introduces "Editing capabilities" later on. I suggest we move the addition of the user detail view after thaht part.